README.md

```diff
@@ -1 +1,10 @@
 Website for Compose Conference. Please visit us at [www.composeconference.org](http://www.composeconference.org)...
+
+
+### Contributions
+- Fork this repo
+- Start in various terminals:
+    - `bundle && bundle exec jekyll build --watch`
+    - `cd _site && python3 -m http.server 8000`
```

Comment:
Shouldn't need to do this if you have jekyll...

Reply:
`bundle exec jekyll serve --watch`

Reply:
Nice! I didn’t know that

Reply:
Feel free to amend the comment and push now that you have access :)

Reply:
Oh, I'm not allowed to approve my own PRs. I think you'll have to help me here.